cypress/README.md

```diff
@@ -35,6 +35,24 @@ See best [the docs on best practice for selectors](https://docs.cypress.io/guide
 
 Follow [Cypress best practice](https://docs.cypress.io/guides/references/best-practices)
 
+### How to use cypress-autorecord
+To allow for auto-recording and stubbing to work, require `cypress-autorecord` in each of your test file and call the function at the beginning of your parent describe block.
+
+```js
```

Comment:
As we use Typescript, can you update this to `ts` and update the code style to match our lint? E.g. remove `;`.